src/elements/paginator/paginator.ts

```diff
@@ -129,6 +132,15 @@ export class SbbPaginatorElement extends SbbNegativeMixin(LitElement) {
     }
   }
 
+  protected override firstUpdated(_changedProperties: PropertyValues): void {
+    super.firstUpdated(_changedProperties);
+
+    this.shadowRoot!.querySelector('sbb-select')?.setAttribute(
```

Comment:
firstupdate is not enough. whenever you activate the options at a later point, the connection should be set as well. the same for when you remove it (maybe not very bad, as a non reachable aria-label will not cause any problems).

Reply:
The logic has been moved to the `updated` hook